plugins/TodayAtMun/__init__.py

```diff
@@ -30,9 +29,13 @@ async def today_next(self, ctx: commands.Context):
         self.today_fun.find_event(self.today_fun.date)
         embed = discord.Embed(
             title=self.today_fun.formatted_date,
-            description=f"```{self.today_fun.diary[self.today_fun.key]}``` ( !today later ) to get next event",
+            description=f"`{self.today_fun.diary[self.today_fun.key]} ( !today later ) to get next event`",
             url=self.parse.DATA_SOURCE,
-            colour=discord.Colour.orange(),
+            colour=discord.Colour.dark_green(),
+        )
+        embed.set_footer(
+            text="\u200b",
+            icon_url="https://raw.githubusercontent.com/MUNComputerScienceSociety/csclub-homepage/master/listing_cs_logo.png",
```

Comment:
nice usage of the cs logo frmo the csclub-homepage repo! i would not have thought to pull it in from there!